src/frontend/get-github-configuration.ts

```diff
@@ -1,59 +1,63 @@
 import { Installation, Subscription } from "../models";
 import { NextFunction, Request, Response } from "express";
-import { getInstallation } from "./get-jira-configuration";
+import { getInstallations, InstallationResults } from "./get-jira-configuration";
```

Comment:
/nit Should we move getInstallations and InstallationResults somewhere in util class (or closer to the model) so our APIs code won't depend on each other?

Reply:
Agreed.  I'm working on another PR to fix those kinds of issues as well as some type stuff.

Reply:
agreed, I'll have another PR to address this.